pkg/orchestrator/gitHubActions.go

```diff
@@ -103,3 +184,33 @@ func isGitHubActions() bool {
 	envVars := []string{"GITHUB_ACTION", "GITHUB_ACTIONS"}
 	return areIndicatingEnvVarsSet(envVars)
 }
+
+func (g *GitHubActionsConfigProvider) GetStageIds() ([]int, error) {
```

Comment:
Does GetStageIds needs to be public? Otherwise:
```suggestion
func (g *GitHubActionsConfigProvider) getStageIds() ([]int, error) {
```